cmd/kyma/alpha/delete/cmd.go

```diff
@@ -43,6 +43,7 @@ func NewCmd(o *Options) *cobra.Command {
 	cobraCmd.Flags().DurationVarP(&o.Timeout, "timeout", "", 1200*time.Second, "Maximum time for the deletion")
 	cobraCmd.Flags().DurationVarP(&o.TimeoutComponent, "timeout-component", "", 360*time.Second, "Maximum time to delete the component")
 	cobraCmd.Flags().IntVar(&o.Concurrency, "concurrency", 4, "Number of parallel processes")
+	cobraCmd.Flags().BoolVarP(&o.KeepCRDs, "keep-crds", "", false, "Keep CRDs during deletion.")
```

Comment:
We should be consistent and match the other flags' descriptions in structure and style.
```suggestion
	cobraCmd.Flags().BoolVarP(&o.KeepCRDs, "keep-crds", "", false, "Flag specifying whether to keep CRDs during deletion")
```